showcase/templates/view_showcase.html

```diff
@@ -0,0 +1,71 @@
+{% extends "base.html" %}
+{% load static %}
+{% load teams_tags %}
+{% load account_tags %}
+
+{% block extra_meta %}
+
+<meta property='og:title' content='{{showcase.hack_project.display_name}}'/>
```

Comment:
XHTML self-closing tags not required in HTML `/>`

Reply:
Removed the / at the end of the self-closing tags